python/cudf/cudf/core/column/string.py

```diff
@@ -46,13 +43,17 @@ def str_to_boolean(column: StringColumn):
 
 
 if TYPE_CHECKING:
+    import cupy
+    from numba import cuda
```

Comment:
Same as above -- let's try to use `import numba.cuda`.

Reply:
Sounds good, changed